### DISCUSSION TITLE
I am unable to install Wix toolset version v3.11.2, how do I resolve this issue?

### DISCUSSION THREAD

**Discussion by hgutierrez-stantec** [wixtoolset/issues#6860] [Questions]
## Triage

Issues are triaged at [online meetings](https://www.firegiant.com/blog/), generally held alternate Thursdays at 9:30 a.m. Pacific time (UTC-7/UTC-8). Meeting requests are sent to the [wix-devs](http://wixtoolset.org/documentation/mailinglist/#wix-devs) and wix-users mailing lists, as well as in Discussions.

## Discussions and support

Note that discussions are best held on the mailing lists not in issues. Please do not open issues requesting support or debugging help; first start a thread on the [wix-users](http://wixtoolset.org/documentation/mailinglist/#wix-users) mailing list and open an issue if the discussion suggests a bug in the WiX Toolset or that a feature request is appropriate. You can also try out the beta [Discussions](https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions) feature.

## Instructions

Read the sections above. Delete the Triage, Discussions and support, and Instructions sections and fill out the rest.

## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> (WiX version in major.minor.release.build format)

* Which version of Visual Studio are you building with (if any)?

> (Visual Studio version, including year and update)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> (WiX Toolset Visual Studio Extension version in major.minor.release.build format)

* Which version of .NET are you building with?

> .NET 4

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Windows 10

* Describe the problem and the steps to reproduce it.

> I downloaded the wix toolset from this link here: https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm. I simply downloaded "wix311.exe", ran the installer as admin, and clicked on the "Install" button. It tries for a few minutes, and then the installer just says that the install "Failed. Try again?". I don't have any error popups appearing.

I'll be using this as a VS solution that I need to open and build requires an install of the wix toolset to build it.

* Describe the behavior you expected and how it differed from the actual behavior.

> (Be as specific as you can and err on the side of providing too much information.)

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by barnson** [comment]
Check for logs in your %TEMP% directory.

**Reply by hgutierrez-stantec** [reply]
I found a Wix .log file in the "%TEMP%" directory and there seemed to not be anything to note. The only lines I found labelled errors are:

Error 0x800705b4: Failed to wait for child to connect to pipe.
[719C:A22C][2022-08-08T13:52:37]e000: Error 0x800705b4: Failed to connect to elevated child process.
[719C:A22C][2022-08-08T13:52:37]e000: Error 0x800705b4: Failed to actually elevate.
[719C:A22C][2022-08-08T13:52:37]e000: Error 0x800705b4: Failed to elevate.

**Comment by rseanhall** [comment]
0x800705b4 is ERROR_TIMEOUT. Sounds like an antivirus product is interfering. Or the UAC prompt is flashing on the taskbar.
